AutomatedTesting/Gem/PythonTests/editor_test_testing/TestSuite_Main.py

```diff
@@ -28,10 +30,12 @@ def get_editor_launcher_platform():
         return None
 
 
+# Other plugins can create cross-object reference issues due these tests executing nonstandard pytest-within-pytest
+@mock.patch('ly_test_tools._internal.pytest_plugin.terminal_report._add_ownership', mock.MagicMock())
```

Comment:
A confounding surprise while debugging a fresh failure on Python 3.10: Somehow interacting with a pathlib.Path object in a pytest plugin was able to cause invalid results in pytest internal `_pytest.pytester.RunResult`. This only reproduced in this module, which runs EditorTest via pytest inside another pytest instance.

Reply:
There are apparently existing issues with this file: https://github.com/o3de/o3de/issues/11528